.circleci/Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM circleci/rust:1.40.0
+# Build with `docker build -t jamwaffles/circleci-embedded-graphics:<rust_version> -f ./.circleci/Dockerfile .`
+# Push with `docker push jamwaffles/circleci-embedded-graphics:<rust_version>`
 
-RUN sudo apt update && sudo apt install -y libsdl2-dev && sudo apt clean
+FROM cimg/rust:1.40.0
 
-RUN cargo install cargo-readme just cargo-deadlinks
+RUN sudo apt update && sudo apt install -y libsdl2-dev libssl-dev && sudo apt clean
+
+RUN cargo install just --version ^0.6
```

Comment:
Do newer `just` versions have problems with rust `1.40` or why do you explicitly specify a version for `just`?
Maybe we should specify the version for all cargo tools we use to make the build reproducible.

Reply:
Yeah they do, due to use of the `matches!` macro. I've pushed a commit to pin the other dependencies installed in the image (and replaced the image on Docker Hub under the same tag). I decided to allow patch releases so they're not perfectly pinned. Let me know if you think this is a bad idea and I'll change them to `=x.y.z`.